frontend/app/src/components/inputs/AmountInput.vue

```diff
@@ -1,9 +1,3 @@
-<script lang="ts">
-export default {
-  inheritAttrs: false
-};
-</script>
-
```

Comment:
So I think we can remove this completely since we bind it manually too
`v-bind="useAttrs()"`

I remember why I set the `inheritAttrs: false` though, it was to prevent prop `type` to be passed to the `v-text-field`, but now I try to pass the type, it doesn't cause an error anymore, so I think it is safe to remove this.